### PR TITLE
Updates for Helm v3.8.0 release

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: content/en/docs/topics/registries.md [hunk 1]
```diff
@@ -11,17 +11,16 @@ stored and shared across OCI-based registries.
 
 ## Enabling OCI Support
 
-Currently OCI support is considered *experimental*.
+Prior to Helm v3.8.0, OCI support was considered *experimental* and needed to be
+enabled. As of v3.8.0 it is enable by default.
```
Comment (bridgetkromhout) [inline content/en/docs/topics/registries.md:15]: > As of v3.8.0 it is enable by default.

As of v3.8.0 it is enable**d** by default.

File: content/en/docs/topics/registries.md [hunk 2]
```diff
@@ -11,17 +11,16 @@ stored and shared across OCI-based registries.
...
 
-In order to use the commands described below, please set `HELM_EXPERIMENTAL_OCI`
-in the environment:
+To enable the OCI experiment for Helm versions prior to v3.8.0, please set
```
Comment (bridgetkromhout) [inline content/en/docs/topics/registries.md:17]: > To enable the OCI experiment for Helm versions prior to v3.8.0

Maybe we want to say "To enable OCI experimental support for Helm versions prior to v3.8.0"? This is fine as it is, but my edit may make it a little clearer.

File: content/en/docs/helm/helm.md [hunk 3]
```diff
@@ -89,6 +89,8 @@ By default, the default directories depend on the Operating System. The defaults
 * [helm package](helm_package.md)	 - package a chart directory into a chart archive
 * [helm plugin](helm_plugin.md)	 - install, list, or uninstall Helm plugins
 * [helm pull](helm_pull.md)	 - download a chart from a repository and (optionally) unpack it in local directory
+* [helm push](helm_push.md)	 - push a chart to remote
+* [helm registry](helm_registry.md)	 - login to or logout from a registry
```
Comment (bridgetkromhout) [inline content/en/docs/helm/helm.md:93]: I think the build is breaking due to not finding these two files?

File: content/en/docs/topics/registries.md (hunk 2 above)
Comment (mattfarina) [reply content/en/docs/topics/registries.md:17]: done

File: content/en/docs/topics/registries.md (hunk 1 above)
Comment (mattfarina) [reply content/en/docs/topics/registries.md:15]: done

File: content/en/docs/helm/helm.md (hunk 3 above)
Comment (mattfarina) [reply content/en/docs/helm/helm.md:93]: missed a git add. Thanks.

**Review by bridgetkromhout — APPROVED** [review]
LGTM

**Pull request merged**